virtualfish/virtual.fish

```diff
@@ -442,19 +445,44 @@ function __vf_help --description "Print VirtualFish usage information"
 end
 
 function __vf_globalpackages --description "Toggle global site packages"
-  if set -q VIRTUAL_ENV
-      # use site-packages/.. to avoid ending up in python-wheels
-      pushd $VIRTUAL_ENV/lib/python*/site-packages/..
-      if test -e $VIRTUALFISH_GLOBAL_SITE_PACKAGES_FILE
-        echo "Enabling global site packages"
-        command rm $VIRTUALFISH_GLOBAL_SITE_PACKAGES_FILE
-      else
-        echo "Disabling global site packages"
-        touch $VIRTUALFISH_GLOBAL_SITE_PACKAGES_FILE
-      end
-      popd
-    else
-        echo "No virtualenv is active."
+    if set -q VIRTUAL_ENV
+	pushd $VIRTUAL_ENV
+
+	# if pyvenv.cfg is present, toggle configuration value therein
+	# (https://www.python.org/dev/peps/pep-0405/#isolation-from-system-site-packages)
+	#
+	# otherwise use legacy no-global-site-package.txt file in lib/python*
+
+	if test -e $VIRTUALFISH_VENV_CONFIG_FILE  # PEP 405
+	    # toggle
+	    sed -i '/include-system-site-packages/ {s/true/false/;t;s/false/true/}' $VIRTUALFISH_VENV_CONFIG_FILE
+
+	    # read new state
+	    if [ "true" = (sed -n 's/include-system-site-packages\s=\s\(true\|false\)/\1/p' $VIRTUALFISH_VENV_CONFIG_FILE) ]
+		set enabled 0
```

Comment:
For consistency, it's probably best to indent lines after `if` / `else` statements, as has been done elsewhere in this file (and indeed, in other parts of this PR as well). I realize this results in a lot of indentation with nested `if` statements, but I think it still ends up being significantly more readable.

Reply:
Strange. My editor showed this correctly. Tried to fix this in e82fb6a (output of `cat` is correct). Is it looking good to you as well?

Reply:
It still appears improperly-indented here: https://github.com/justinmayer/virtualfish/blob/e82fb6a5fa1dd8f5c6c5a36a8625e02a1dc6d7de/virtualfish/virtual.fish#L450-L451

On the other hand, it seems to be properly indented when looking at the same line in [raw mode](https://raw.githubusercontent.com/justinmayer/virtualfish/e82fb6a5fa1dd8f5c6c5a36a8625e02a1dc6d7de/virtualfish/virtual.fish). Drag-selecting the white space to the left indicates that it may be one or more tabs instead of spaces. What editor are you using? Is your editor perhaps adding tabs rather than spaces when you press the Tab key?

Reply:
Thanks for the hint. Is hopefully fixed in 1842637. I re-indented the whole function.